scripts/snapshot-docdb.sh

```diff
@@ -1,6 +1,23 @@
 #! /bin/bash
 set -e
 
+# Snapshot ID
+SSID=dbss-$(date +"%s")
+
+# Create Snapshot
 aws docdb create-db-cluster-snapshot \
-  --db-cluster-identifier ${DOCDB_CLUSTER_IDENTIFIER} \
-  --db-cluster-snapshot-identifier dbss-${IMAGE_TAG}
\ No newline at end of file
+  --db-cluster-identifier portal-client \
+  --db-cluster-snapshot-identifier $SSID --tags Key=sha,Value=${IMAGE_TAG}
+
+# DocDB Snapshot
+DBSS=$(aws docdb describe-db-cluster-snapshots --db-cluster-snapshot-identifier $SSID |jq -r '.DBClusterSnapshots[]' | jq -r '.Status')
+
+# Don't do anything else until it's available
+# aws docdb does not have a `wait`
+until [ "$DBSS" = "available" ]
+    do
+      DBSS=$(aws docdb describe-db-cluster-snapshots --db-cluster-snapshot-identifier $SSID |jq -r '.DBClusterSnapshots[]' | jq -r '.Status')
+      echo $DBSS
+    done
+    echo "$SSID available!"
+    continue
```

Comment:
Bash scripts aren't my strong suit so forgive the dumb question, but do you need a new line at the end of the file?

Reply:
I'm not sure it matters. Do you have a linter going off?

Reply:
I installed shellcheck and it doesn't complain about the lack of a new line.

Reply:
No I'm not seeing any errors, I was just curious. Looking in the GitHub UI it has that little red circle and I just thought that was a warning or something.

Reply:
I hadn't noticed that. Good catch.